### ISSUE TITLE
Bug Report: 

### ISSUE THREAD

**Issue by Fancy2209** [FunkinCrew/Funkin#356]
# Please check for duplicates or similar issues, as well performing simple troubleshooting steps (such as clearing cookies, clearing AppData, trying another browser) before submitting an issue.

### Web
- [ ] [Newgrounds](https://www.newgrounds.com/portal/view/770371)
- [ ] [Itch.io](https://ninja-muffin24.itch.io/funkin)

#### If you are using a web build, what browser are you using?
- [ ] Chrome, or a Chromium based broswer (Edge, Opera, etc.)
- [ ] Firefox, or a Gecko based broswer
- [ ] Safari
- [ ] Other, please specify:

### Local
- [x] Windows x86
- [ ] Windows x86_64
- [ ] Linux
- [ ] macOS

## What version of the game are you using? Look in the bottom left corner of the main menu.
0.2.7
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
No sorry, this just happens 
## Please describe your issue. Provide extensive detail and images if possibe.
Basically when I try to play the game there are no characters, just blacksquares (Im reporting this issue because everyone else who tried didn't provide info)
here are some images;
![image](https://user-images.githubusercontent.com/64917206/106947890-88640900-671a-11eb-9816-1416661814b1.png)
![image](https://user-images.githubusercontent.com/64917206/106947840-71bdb200-671a-11eb-9c67-84972b46a1d0.png)
![image](https://user-images.githubusercontent.com/64917206/106947971-aa5d8b80-671a-11eb-8c36-feeb90a53a89.png)
![image](https://user-images.githubusercontent.com/64917206/106948030-bf3a1f00-671a-11eb-8fc8-670d4fe6ea4c.png)

**Comment by Fancy2209** [comment]
Here are the PC specs if it helps btw
![image](https://user-images.githubusercontent.com/64917206/106948149-e0027480-671a-11eb-9b0c-2bb44f6a2f1c.png)

**Comment by Midou36O** [comment]
Check #370
